<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MrqAxesPage</class>
 <widget class="QWidget" name="MrqAxesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="mrqAxes" name="pageMotor">
      <attribute name="title">
       <string>Motor</string>
      </attribute>
     </widget>
     <widget class="mrqAxes2" name="pageMotion">
      <attribute name="title">
       <string>Motion</string>
      </attribute>
     </widget>
     <widget class="MrqTrigPage" name="pageTrigger">
      <attribute name="title">
       <string>POS</string>
      </attribute>
     </widget>
     <widget class="MrqTune" name="pageTune">
      <attribute name="title">
       <string>Tune</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mrqAxes</class>
   <extends>QWidget</extends>
   <header>mrqaxes.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mrqAxes2</class>
   <extends>QWidget</extends>
   <header>mrqaxes2.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MrqTrigPage</class>
   <extends>QWidget</extends>
   <header>mrqtrigpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MrqTune</class>
   <extends>QWidget</extends>
   <header>mrqtune.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
